src/Sulu/Component/Webspace/Analyzer/AdminRequestAnalyzer.php

```diff
@@ -179,7 +179,7 @@ public function getCurrentPostParameter()
      * Returns the get parameters
      * @return array
      */
-    public function getCurrentGetParameter()
+    public function getGetParameters()
```

Comment:
Why do we do this? It is wrong

These parameters are only used by the ExceptionHandler as far as I can tell.